### PR TITLE
feat(swap-form): add swap form in market page

### PR DESCRIPTION
**Pull request by shmodarresi**

### CODE REVIEW

File: src/contracts/Market.ts [hunk 1]
```diff
@@ -216,6 +216,8 @@ export const sellTokens = async (
     'unit',
     amount,
   );
+  console.log(`amount: ${amount} , aToSwap: ${toSwap} , swapSlippage: ${swapSlippage}`);
```
Comment (tlebon) [inline src/contracts/Market.ts]: remove this please

File: src/design-system/organisms/SwapForm/SwapForm.tsx [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+        ];
+        setExpectedSwap(newExpectedSwap);
+
+        if (connected) {
+          const [newYes, newNo, yesOpt, noOpt] =
+            TokenType.yes === swapTokenType
+              ? [quantity, swapOutput, '-', '+']
+              : [swapOutput, quantity, '+', '-'];
+
+          let newTotalValue;
```
Comment (tlebon) [inline src/design-system/organisms/SwapForm/SwapForm.tsx]: i think i prefer this as a ternary expression using a const for setting either value depending on the tokenType, but I'm not sure if its 'better'

Comment (tlebon) [reply src/design-system/organisms/SwapForm/SwapForm.tsx]: example: 
```suggestion
          const newTotalValue= TokenType.yes? tokenPrice.yes * (userAmounts.yesToken - newYes) +
              tokenPrice.no * (userAmounts.noToken + newNo):tokenPrice.yes * (userAmounts.yesToken + newYes) +
              tokenPrice.no * (userAmounts.noToken - newNo);
```

Comment (tlebon) [reply src/design-system/organisms/SwapForm/SwapForm.tsx]: or better as a function with tokenType as a parameter as you suggested!

**Comment by tlebon** [comment]
Screen Shot 2021-09-03 at 14 34 26
rounding should be improved as this is unclear

**Comment by tlebon** [comment]
Screen Shot 2021-09-03 at 14 34 02
maybe we can flex this so its a row instead of wrapping

File: src/design-system/organisms/SwapForm/SwapForm.tsx [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+  const [pools, setPools] = React.useState({
+    yesPool: 0,
+    noPool: 0,
+  });
+  const [swapToken, otherToken] = TokenType.yes === swapTokenType ? ['Yes', 'No'] : ['No', 'Yes'];
+  const PositionDescription = () => (
+    <>
+      {t('expectedAdjustedPosition')}
+      <IconTooltip
+        description="Swap rate is discounted by 0.3% fee. This fee goes to the liquidity providers."
```
Comment (tlebon) [inline src/design-system/organisms/SwapForm/SwapForm.tsx]: move this to a translation i think

**Review by shekhar-shubhendu — APPROVED** [review]
LGTM

**Pull request merged**